.gitignore

```diff
@@ -33,3 +33,5 @@
 
 # VSCode build output.
 out/
+data/
```

Comment:
Nope. The `data/` directory must not be ignored. We continue to have the data download/creation scripts in there...

Reply:
Ah, oops, I do this locally everytime, since otherwise code searches in vscode take ages. Will revert

Reply:
Done